projects/coinweb/index.js

```diff
@@ -0,0 +1,11 @@
+const { staking } = require('../helper/staking');
+
+module.exports = {
+  ethereum: {
+    tvl: () => ({}),
+    staking: staking(
+      '0x13Fe7160858F2A16b8e4429DFf26c8a3A4b12b1B',
```

Comment:
this is not a staking contract? 
![image](https://github.com/user-attachments/assets/82381ec6-ee4f-457c-b234-1dd12b9c98cd)

Reply:
Hello @g1nt0ki , vesting contract.

Reply:
sorry, we dont track vesting atm.